plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -308,6 +308,7 @@ private function init_hooks() {
 		add_action( 'woocommerce_installed', array( $this, 'add_woocommerce_remote_variant' ) );
 		add_action( 'woocommerce_updated', array( $this, 'add_woocommerce_remote_variant' ) );
 
+		add_filter( 'robots_txt', array( $this, 'robots_txt' ) );
```

Comment:
The class is already using the `AccessiblePrivateMethods` trait, so we could use `self::add_filter` here instead and make the `robots_txt` method private.

Reply:
👍 Updated in 9505e70269